src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -1115,7 +1115,8 @@ public static TheoryData<bool> UseShellExecute
                 TheoryData<bool> data = new TheoryData<bool> { false };
 
                 if (   !PlatformDetection.IsUap // https://github.com/dotnet/corefx/issues/20204
-                    && !PlatformDetection.IsWindowsNanoServer) // By design
+                    && !PlatformDetection.IsWindowsNanoServer // By design
+                    && !PlatformDetection.IsWindowsIoTCore)
```

Comment:
The previous line says "By design"... is this one by design, too?

Reply:
Yes